recipes/radioactivedecay/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "radioactivedecay" %}
+{% set version = "0.3.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/radioactivedecay-{{ version }}.tar.gz
+  sha256: ddb0a1054cf659db465a7355be3f8ac203e81206aa90b07798ac0eeafb65d99a
+
+build:
+  number: 0
+  skip: true   # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - importlib_resources  # [py<37]
```

Comment:
In this case it's best to just always depend on `importlib_resources`.  Or you can remove it entirely and just make the dependencies on Python as `>=3.7`. You'll need to at least make them `>=3.6` when making the recipe noarch.

Reply:
Thanks, opted for the second option so the allowed Python versions (>=3.6) are the same on PyPI and on conda-forge.